CHANGELOG.md

```diff
@@ -34,6 +34,7 @@
 ## stream-chat-android-ui-components
 - Add MessagesHeaderView
 - New artifact name: `io.getstream:stream-chat-android-ui-components:STREAM_VERSION`
+- Adding AttachmentGallery to handle attachments
```

Comment:
It seems that you added changelog on wrong place

Reply:
We could avoid merge/rebase problems in the changelog if the "to be released" parts were kept in a separate file, and moved to the actual changelog when the release is created. What do you think, @JcMinarro?

Reply:
Damn... Sorry merge problem.

Reply:
I don't really see a problem with this behavior. Having it into a separated file could be the same, add some changes that are previously added in other version or totally remove the new changes added to the "to be released" file.
We can discuss if we would prefer a new file or any other alternative